index.js

```diff
@@ -38,8 +38,17 @@ function installXgboost(artifacts_version, xgboost_version) {
 }
 
 function installLibbpf() {
-  core.info(`libbpf`);
-  executeCommand("sudo apt-get install -y libbpf-dev", "fail to install libbpf related package");
+  core.info(`installing libbpf v1.2.0 from source`);
```

Comment:
get the libbpf version from env (as other pkgs)

Reply:
updated

Reply:
can you pls run the tests?